src/app/src/main/kotlin/org/imperial/mrc/hint/security/tokens/OneTimeTokenManager.kt

```diff
@@ -16,7 +17,7 @@ class OneTimeTokenManager(
         appProperties: AppProperties,
         private val tokenRepository: TokenRepository,
         signatureConfiguration: SignatureConfiguration,
-        private val authenticator: OneTimeTokenAuthenticator
```

Comment:
Why did this change?

Reply:
Ah this isn't actually relevant until the next PR. `OneTimeTokenAuthenticator` has a more complicated constructor than the class it inherits from. The UserCLI doesn't actually do any authentication, so no need for any custom logic there, we may as well just pass the base class which is easier to instantiate. https://github.com/mrc-ide/hint/pull/148/files#diff-7083312459829a5d01f15b4f2e7dd398R109

Reply:
Oh, so can we get rid of OneTimeTokenAuthenticator then? Doesn't get used anywhere else, except  as a mock in OneTimeTokenManagerTests (which I guess should mock a JwtAuthenticator instead now).

Reply:
Isn't it essential for verifying one time tokens?? The main app still has to do that!

Reply:
It should still be being wired up as a bean and injected by spring, just not used in the UserCLI where we're doing the dep injection manually.

Reply:
Actually the more correct solution here would be to split out the `OnetimeTokenManager` into a `OnetimeTokenIssuer` and `OnetimeTokenAuthenticator`, then the `EmailManager` could just take the former as a dependency. Although I'm somewhat concerned that we need to get these new users added asap, so I'm minded to leave as is for now and do that refactor on a separate branch.

Reply:
>It should still be being wired up as a bean and injected by spring
Ok, because it's the only implementation of JwtAuthenticator that's got @Configuration on it, so that's the one which will be instantiated when it creates the parameters for the OneTimeTokenManager.

Reply:
Exactly :+1: But I will add a ticket for refactoring into separate Issuer and Authenticator